src/components/ListOfWinners.js

```diff
@@ -8,21 +8,35 @@ const ListOfWinners = () => {
   return (
     <div className="winners">
       <div className="winners__card">
-        <div className="winners__img-container">
+        <div className="winners__header">
+          <h2>Ranking</h2>
           <img height="40"
                alt="winners"
                className="winners__medal"
                src="https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/thumbs/240/google/223/sports-medal_1f3c5.png" />
         </div>
         <div className="winners__container">
           {winnersList.length > 0 ?
-            winnersList.map((winner, index) =>
-            <div key={index}
-                 className="winners__winner">
-              <div>{winner.name}</div>
-              <div>{winner.score}</div>
-              <img src={winner.avatarUrl} />
-            </div>)
+              <table className="winners__table">
+                <thead>
+                <tr>
+                  <td>Drużyna</td>
+                  <td>Wynik</td>
+                </tr>
+                </thead>
+                <tbody>
+                {winnersList.map((winner, index) => (
```

Comment:
Generally speaking, array index shouldn't be used as `key` property, but it shouldn't be a problem in this case.